torch/distributed/rpc/api.py

```diff
@@ -58,6 +59,29 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+# States used by `def _wait_all_workers()`.
+_ALL_WORKER_NAMES = None
+_INTENDED_WORKER_NAMES = set()
+_PROCEED_SIGNAL = threading.Event()
```

Comment:
Add some comments here about what each of these global variables are storing.

Reply:
Adding.